src/coreclr/vm/ilstubcache.cpp

```diff
@@ -545,6 +547,12 @@ MethodDesc* ILStubCache::GetStubMethodDesc(
             pContainingModule = pTargetMD->GetLoaderModule();
```

Comment:
Can this be:

if (pSigLoaderModule == NULL)
{
    pSigLoaderModule = (pTargetMD != NULL) ? pTargetMD->GetLoaderModule() : pSigModule;
}
```
and use `pSigLoaderModule` instead of `pContainingModule` in the rest of the method.